library/common/http/client.cc

```diff
@@ -176,6 +176,19 @@ void Client::DirectStream::resetStream(StreamResetReason reason) {
   callbacks_->onError();
 }
 
+void Client::DirectStream::readDisable(bool disable) {
+  if (disable) {
+    ++read_disable_count_;
+  } else {
+    ASSERT(read_disable_count_ > 0);
+    --read_disable_count_;
+    if (read_disable_count_ == 0 && wants_write_notification_) {
+      //      bridge_callbacks_.on_can_send_data(bridge_callbacks_.context);
+      wants_write_notification_ = false;
```

Comment:
I wonder if this should be done before the call to `on_can_send_data()` so that if `on_can_send_data()` ends up calling `sendData()` we don't then end up clearing `wants_write_notification_` erroneously?